db/migrate/20200302180757_copy_price_data.rb

```diff
@@ -0,0 +1,9 @@
+class CopyPriceData < ActiveRecord::Migration[6.0]
```

Comment:
If you're not using change and it's not a migration that can be reversed [the docs](https://edgeguides.rubyonrails.org/active_record_migrations.html#using-the-up-down-methods) say you should raise the proper exception in your `down` method.

Not sure if this is something that can be reversed if the column `price` gets dropped, so maybe use `change` or specify an appropriate `down` behaviour?